### DISCUSSION TITLE
Run your solution on scale for Knowlege search in a big company

### DISCUSSION THREAD

**Discussion by osi1880vr** [llmware-ai/llmware#264] [Q&A]
Hello,

im working in a quite large company inside the insurance industry, we are planing on setting up an AI assistant which we want to feed with our very large base on business know how as well as all the stuff we did document in the IT department but is hard to find anything anymore as all the expensive enterprise tools only provide keyword search. 

so the question is will your RAG solution scale when connected to a large filled vector database? Im asking as we have a hard time to impelemt GPU power into our system but we would have quite a bit in spare CPU time.
Lets say we talk about a constant number of 50 users talking to the system at the same time.

How about your solution together with langchain? Any way that would work?

thanks a lot and sorry if I did not find the answer to the same question someone has asked before me =)

Kind regards
Werner

**Comment by NYDocutest** [comment]
Hi Thank you for your question. Our system is definitely set up for scale - we have had experience with scaling libraries with each library containing hundreds of thousands of large documents using our system with multiple concurrent users. Regarding Langchain, we would need to understand the features in Langchain you are referring to. We would be happy to discuss this project in more detail - please feel free to set up a meeting with us for a more in-depth discussion! https://calendly.com/noberst/discovery-call

**Reply by osi1880vr** [reply]
Well this was a very nice answer as well as a very nice and helpfull meeting :) Thank you very much

**Reply by osi1880vr** [reply]
You told me you also got discord to get in contact with you guys. I fail to find it, could you provide a invite link?

**Reply by NYDocutest** [reply]
Hi Werner,
Thank you so much for your kind feedback! I had also sent you 2 follow up emails after our call to send to you the links I referenced during our call. I hope they are helpful. In the meantime, here is our discord link: https://discord.gg/CbKf9Kwz3E
Please let me know how your experimentation goes!
